src/controller_test.py

```diff
@@ -24,33 +24,3 @@ def soil_moisture(self):
                 return 900
```

Comment:
Up at the top of this file, do we still need a MockLight class now we don't test for Lights?

Reply:
We don't even need the controller, so we removed them!